log.go

```diff
@@ -111,11 +109,20 @@ func createLogger(ctx context.Context, event EventIncoming, headers http.Header)
 	labels["cloud_trace_context"] = headers.Get("X-Cloud-Trace-Context")
 	labels["trace_parent"] = headers.Get("traceparent")
 
-	if projectID != "" {
-		client, _ = logging.NewClient(ctx, projectID)
-		gcpLogger = client.Logger("skill_logging")
+	return labels
+}
+
+func createGcpLogger(ctx context.Context, labels map[string]string) *Logger {
+	if projectID == "" {
+		return nil
 	}
 
+	var gcpLogger *logging.Logger
+	var client *logging.Client
+
+	client, _ = logging.NewClient(ctx, projectID)
```

Comment:
Perhaps you want to check if there's an error here and return `nil` if the GCP logging client couldn't be instantiated?

Reply:
I've pushed a change that should maintain the existing behaviour.